src/shared/components/ProfessionCard.tsx

```diff
@@ -21,28 +20,26 @@ const CardHeaderContainer = styled(CardHeader)`
 const AvatarContainer = styled(Avatar)<{ background: ProfessionKey }>`
   margin-right: 1em;
   background-color: ${({ background, theme }) => theme.palette.professions[background].main};
-  padding: .75em;
+  padding: 0.35em;
 `;
 
-type CardProps = {
+type ProfessionCardProps = {
   title: string;
   description: string;
   avatarSrc: string;
   avatarBackground: ProfessionKey;
-  clickHandler: (route: string) => void;
-  route: string;
+  isClickable?: boolean;
 };
 
-export const ProfessionCard: React.FC<CardProps> = ({
+export const ProfessionCard: React.FC<ProfessionCardProps> = ({
   title,
   description,
   avatarSrc,
   avatarBackground,
-  clickHandler,
-  route,
+  isClickable,
 }) => (
-  <CategoryCardContainer onClick={() => clickHandler(route)}>
+  <ProfessionCardContainer $isClickable={isClickable !== undefined}>
     <CardHeaderContainer title={title} subheader={description} />
     <AvatarContainer src={avatarSrc} background={avatarBackground} />
-  </CategoryCardContainer>
+  </ProfessionCardContainer>
 );
```

Comment:
I use here - 'transient props' - https://github.com/styled-components/styled-components/releases/tag/v5.1.0
It solves problem with passing props that are needed only for styling purpose - not for whole component.